### PR TITLE
[DSOUND] Sync with Wine 3.2 + Wine-Staging 2.21

### PR DESCRIPTION
**Pull request by tkreuzer**

### CODE REVIEW

File: sdk/include/psdk/dsound.h [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
- * This library is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public
- * License along with this library; if not, write to the Free Software
- * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
- */
+* Copyright (C) the Wine project
```
Comment (ThFabba) [inline sdk/include/psdk/dsound.h]: This isn't right, Wine has those spaces.

File: sdk/include/psdk/dsound.h [hunk 2]
```diff
@@ -33,82 +33,104 @@ extern "C" {
 
 #ifndef DX_SHARED_DEFINES
 
-typedef float D3DVALUE, *LPD3DVALUE;
+    typedef float D3DVALUE, *LPD3DVALUE;
```
Comment (ThFabba) [inline sdk/include/psdk/dsound.h]: Wine also doesn't have this indent. Did this file get caught in some auto-formatter?

**Comment by tkreuzer** [comment]
This breaks sound

**Comment by Mouvedia** [comment]
Will this ever be attempted again ?

**Pull request closed without merging**